pkg/controllers/nodeclaim/garbagecollection/suite_test.go

```diff
@@ -290,9 +290,9 @@ var _ = Describe("GarbageCollection", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 	It("should not delete an instance if it was not launched by a NodeClaim", func() {
-		// Remove the "karpenter.sh/managed-by" tag (this isn't launched by a machine)
+		// Remove the "karpv1.NodePoolLabelKey" tag (this isn't launched by a machine)
```

Comment:
should this be adjusted?
```suggestion
		// Remove the "karpenter.sh/nodepool" tag (this isn't launched by a machine)
```